pkg/snapstore/oss_snapstore.go

```diff
@@ -224,6 +219,14 @@ func (s *OSSSnapStore) List(_ bool) (brtypes.SnapList, error) {
 	prefix := path.Join(strings.Join(prefixTokens[:len(prefixTokens)-1], "/"))
 
 	var snapList brtypes.SnapList
+	var bucketImmutableExpiryTimeInDays *int
+
+	wormCfg, err := s.client.GetBucketWorm(s.bucketName)
+	if err != nil {
+		logrus.Warnf("unable to get worm configuration for bukcet: %v", err)
```

Comment:
Is there a specific reason we’re logging the error from GetBucketWorm instead of returning it?

Reply:
As discussed offline, I have now handled the error using error code, PTAL